src/Umbraco.Core/StringExtensions.cs

```diff
@@ -541,7 +541,7 @@ public static T EnumParse<T>(this string strType, bool ignoreCase)
         /// <returns>Returns the string without any html tags.</returns>
         public static string StripHtml(this string text)
         {
-            string pattern = "[*{}\\/:<>?|\"-()\\n]";
+            const string pattern = @"<(.|\n)*?>";

```

Comment:
Hi @elit0451 

I think this is a breaking change.

Let's assume this html snippet

`<h1>My First Heading</h1>`

With the old regex the function would return : h1My First Headigh1 (it also removes the "n"character)

With the new regex this will result in : My First Heading

So the old implementation is actually "broken" and the new one works as expected.

Maybe you can discuss this internally

Reply:
I just see that on latest dev-v7 the regex is actually `<(.|\n)*?>`

Could it be that you didn't start this PR from the latest dev-v7 branch but from the branch from this PR #3814

Reply:
Hi @dawoe,

The case was that I've made some changes to the dev-v7 and then reverted them (one change was left out- the one with the StripHtml()). Afterwards, I've created the branch with from dev-v7 but it happened that it still had that change -> but now it is all reverted